src/scene/Scale.js

```diff
@@ -171,6 +173,13 @@ function ordinal(scale, rng, group) {
   if (!scale.invert && spatial) invertOrdinal(scale);
 }
 
+function setPadding(pad, config, defaultPad, parent_id, datum) {
+  if (!parent_id && config && config.scale && config.scale[pad] && !datum._facetID) {
+    return config.scale[pad];
+  }
+  return defaultPad;
+}
+
```

Comment:
@arvind Yes, I believe so.  I had to add that in or else it was padding individual items in group charts.  This is the only place that this is used though.